Unfortunately it seems on travis we get "bin/validate-package-lock.js(19,30): error TS2307: Cannot find module '../package-lock'.", so the json file is not read as expected.

--- a/bin/validate-package-lock.js
+++ b/bin/validate-package-lock.js
@@ -16,6 +16,8 @@ const { red, yellow } = require( 'chalk' );
 /**
  * Internal dependencies
  */
+// Ignore reason: `package-lock.json` exists outside `bin` `rootDir`.
+// @ts-ignore
 const packageLock = require( '../package-lock' );
 
 const dependencies = Object.entries( packageLock.dependencies );

